modules/ROOT/pages/dataweave-cookbook-pluck.adoc

```diff
@@ -0,0 +1,132 @@
+= Extract Key/Value Pairs with Pluck Function
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, transform, transformer, format, dataweave, data weave, output structure, mime type, mime
+
+The following Mule app example shows how to use the DataWeave `pluck` function to extract key/value pairs from a JSON payload and store them into flow variables with variable name as the keys and their corresponding values.
```

Comment:
```suggestion
The following Mule app example shows how to use the DataWeave `pluck` function to extract key-value pairs from a JSON payload and store them into Mule event variables with variable names as the keys and their corresponding values.
```

Reply:
We don't use the term "flow variable" in Mule 4 as far as I know. Could also be "store into a Mule variable" or something like that.